### PR TITLE
View by day feature

### LINKED ISSUE #60: As a expert user, I can view student session timings in a timetable
I can know what time I will be teaching which student.

### PR DESCRIPTION
**Pull request by phyliciachristel**
Hi guys i almost wanted to kms implementing this so pls be nice <3 
idk if i've fully tested for any bugs bc ^ (but i'll do so one day ha) and also lmk if i should move any implementations around 
(like idk if TimeSlot should be under Person)

also @farisanadia i need u to help make the fxml nice <3

### CODE REVIEW

**Comment by farisanadia** [comment]
not the fxml nice

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 1]
```diff
@@ -46,6 +46,14 @@ public void setFullView() {
         model.setFullView();
     }
 
+    public void setDayView() {
+        model.setDayView();
+    }
+
+    public void setListView() {
+        model.setListView();
+    }
```
Comment (cfyjoanna) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:55]: btw I know I added this first but idk if these are even necessary... like you can just do model.set() directly in the method LOL i was gg thru it! can change next time if need

**Review by cfyjoanna — APPROVED** [review]
LGTM! also ty for picking our many typos in other files... u are god's gift to this world

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java (hunk 1 above)
Comment (phyliciachristel) [reply src/test/java/seedu/address/logic/commands/EditCommandTest.java:55]: help i didnt even question it

**Pull request merged**